Acornfile

```diff
@@ -0,0 +1,33 @@
+services: db: image: "ghcr.io/acorn-io/mariadb:v10.11.5-1"
```

Comment:
Use # notation here to always use the latest Acorn service image

Reply:
I think using specific version will be good as using the `#` will gives us the latest DB, and there might be the case that application version don't support the latest DB version and the application might break.